app/src/main/java/tttTDD/Java/ConsoleClient.java

```diff
@@ -2,17 +2,15 @@
 
 import tttTDD.Java.Interfaces.Player;
 
-import java.util.ArrayList;
-import java.util.InputMismatchException;
 import java.util.Scanner;
-import java.util.stream.IntStream;
 
 public class ConsoleClient {
     private PlayerFactory playerFactory = new PlayerFactory();
+    private Display display = new Display();
 
-    private Game game;
-    private final Scanner input;
     private GameConfiguration gameConfig;
+    private final Scanner input;
```

Comment:
now that you have `GameInput` to wrap the Scanner instance, have you considered removing it from the ConsoleClient?